composables/useGraphql.ts

```diff
@@ -21,7 +21,7 @@ export const useQueryParams = ({
 export default function ({
   queryPrefix = '',
   queryName,
-  clientName,
+  clientName = '',
```

Comment:
I'm scared of this change 😨  spent a lot of time on nuxt3

Reply:
I'm getting this error on my code editor if I didn't put that value. Or is it only on my end?

![image](https://github.com/kodadot/nft-gallery/assets/734428/68c131a9-1c49-4bef-8c03-53ea99ebf65f)

to simplify, I think we can remove `useQueryParams` on nuxt3 on `useGraphql` file. afaik that vars is created to try to solve codeclimate issue at that time

Reply:
> I'm getting this error on my code editor if I didn't put that value. Or is it only on my end?

I have it too that's why on nuxt3 branch I need to pass `clientName: client.value` on every `useGraphql` call

> to simplify, I think we can remove `useQueryParams` on nuxt3 on `useGraphql` file. afaik that vars is created to try to solve codeclimate issue at that time

🤔 I'll check that

Reply:
Meanwhile it has been fixed on my side